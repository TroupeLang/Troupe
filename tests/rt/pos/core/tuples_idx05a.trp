(1,2,3).2147483647