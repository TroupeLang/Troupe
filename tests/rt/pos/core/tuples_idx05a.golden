2024-02-14T11:33:50.114Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread b490728b-03f4-4cbb-af4e-349efbca03d4@{}%{}
>> Index out of bounds: tuple (1@{}%{}, 2@{}%{}, 3@{}%{}) does not have length more than 2147483647
